### DISCUSSION TITLE
When Homeassistent turns a device on it can't be turned off using the device.

### DISCUSSION THREAD

**Discussion by jonaswre** [SukramJ/aiohomematic#1832] [Q&A] [answered]
Hi everybody, I recently started to use more of the custom_homematic integration. And I realized that once a device has been turned on using custom_homematic it can't be turned of using the device itself.

I looked at the channel configuration in the ccu3 and realized that custom_homematic turn on all channels but there is only on direct connection to channel 3 to turn it on/off. That means in a normal configuration the device will stay on because only one of the 3 channels need to be on. 

Since is this such a basic features Im sure I'm doing something wrong. Can I configure what channels custom_homematic will turn on?

**Comment by SukramJ** [comment]
No, every channel can be controlled individually.

> Since is this such a basic features Im sure I'm doing something wrong.

Yes.

Which device you are talking about?
How do you turn on .... ?

**Reply by jonaswre** [reply]
Thanks for your anwser. The device im talking about is HmIP-BSM. Im triggering it through a card on my homeassistent dashboard. 

This is what i looks like before its triggered. 
<img width="493" alt="image" src="https://github.com/user-attachments/assets/9ce1ca97-974f-4072-90e0-c266feba0153">

I then execute this action in homeassistent. 
<img width="390" alt="image" src="https://github.com/user-attachments/assets/fedc366c-a232-47f0-ba66-0244499eac20">

It turns on all the channels.
<img width="493" alt="image" src="https://github.com/user-attachments/assets/7ade8d08-c591-4145-ad1c-3908e3fa2b1f">

When I press the button on the device it will only turn off channel 4.
<img width="493" alt="image" src="https://github.com/user-attachments/assets/63b467f6-78f3-4551-83f9-8a5e2de9b700">

Since the default value is an OR connection between the channels 
<img width="727" alt="image" src="https://github.com/user-attachments/assets/b2e63fa7-fd47-4d2e-8d0f-eb53950631ae">

It will stay ON.

**Reply by SukramJ** [reply, marked as answer]
I don't know what your card is triggering, but when i use a default HA entity list, then  only one channel is triggered.
Looks like you card is triggering all active channels of a device.

**Reply by jonaswre** [reply]
you are right i just checked it when you move a device into an area all channels will be triggered. Im not really using any channels except the first one. is there a way to disable channels?

**Reply by jonaswre** [reply]
I just anwsered my own question. Thanks for your quick help.

**Reply by SukramJ** [reply]
2nd  and 3rd channel are disabled by default.

**Reply by jonaswre** [reply]
> 2nd and 3rd channel are disabled by default.

I must have done something weird when adding them. They always have been enabled. Maybe i've migrated from the old integration. This also explains why my system was so slow to turn of all lights or actions like that because it would have taken 3x the traffic.
